tests/pricing_models/test_dataclasses.py

```diff
@@ -43,8 +44,6 @@ def __getitem__(self, key):
     yieldspace: TestResultCalcInGivenOutSuccess
     hyperdrive: TestResultCalcInGivenOutSuccess
 
-    __test__ = False  # pytest: don't test this class
-
 
 @dataclass
 class TestCaseCalcInGivenOutFailure:
```

Comment:
Since we have a types module for these tests, we may as well make `in_` and `out` quantity and delete the duplicated types.

Reply:
will follow up with this